source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,250 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+You can use a text search to retrieve documents that contain a **term**
+or a **phrase** in a specified field. A term is a sequence of characters
+that excludes whitespace characters. A phrase is a sequence of terms
+that with any number of whitespace characters. 
+
+The following sections show you how to perform the following types of
+text searches:
+
+- Search Text by a Term
+- Search Text by a Phrase
+- Search Text with Terms Excluded
+
+If you want to sort by relevance, see our :ref:`Sort Results
+<sorts-crud-text-search>` page.
+
+Sample Documents
+~~~~~~~~~~~~~~~~
+
+The following sections feature examples that run text searches on the
+``fast_and_furious_movies`` collection. Each section uses a variable
+named "collection" to refer to the ``MongoCollection`` instance of the
+``fast_and_furious_movies`` collection. 
+
+The collection contains documents with a title and tags associated
+with one of the several movies that are part of the Fast and Furious
+movie franchise: 
+
+.. code-block:: json
+
+   { "_id": 1, "title": "2 Fast 2 Furious ", "tags": ["undercover", "drug dealer"] } 
+   { "_id": 2, "title": "Fast 5", "tags": ["bank robbery", "full team"] }
+   { "_id": 3, "title": "Furious 7", "tags": ["emotional"] }
+   { "_id": 4, "title": "The Fate of the Furious", "tags": ["betrayal"] }
+
+Text Index
+----------
+
+You must create a **text index** before running a text search. A text
+index specifies the string or string array field on which to run a text
+search. A collection can only contain one text index.
+
+In the following examples, you run text searches on the ``title``
+field in the ``fast_and_furious_movies`` collection. To enable text
+searches on the ``title`` field, create a text index using the
+:ref:`Indexes <index-text-indexes>` builder with the following 
+snippet: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/SearchText.java
+   :language: java
+   :dedent:
+   :start-after: begin textIndex
+   :end-before: end textIndex
+
+Multiple Indexes
```

Comment:
I think "Multiple Indexes" might be slightly misleading since you're explaining here that you need to use a compound index instead of more than one (multiple) index.
Multiple Indexed Text Fields or Compound Text Index might be better.

Reply:
I'm removing this section and redirecting to the Indexes page.